model/flow/chunk.go

```diff
@@ -362,3 +362,23 @@ func stringsToCids(strs []string) ([]cid.Cid, error) {
 	}
 	return cids, nil
 }
+
+// Equals returns true if and only if receiver BlockExecutionDataRoot is equal to the `other`.
+func (b BlockExecutionDataRoot) Equals(other BlockExecutionDataRoot) bool {
+	// Compare BlockID fields
```

Comment:
From my perspective, this is trivially apparent and doesn't need documentation (... didn’t think I’d ever say this 😅 )

```suggestion
```